src/schema/posts.ts

```diff
@@ -824,7 +857,9 @@ export const resolvers: IResolvers<any, Context> = {
     ): Promise<GQLPost> => {
       const post = await ctx.con.getRepository(Post).findOneByOrFail({ id });
       await Promise.all([
-        ensureSourcePermissions(ctx, sourceId, SourcePermissions.Post),
+        ensureSourcePermissions(ctx, sourceId, {
```

Comment:
Why do we call twice the same function? Can we maybe reduce the number of calls?

Reply:
Oh. Merge issues. Let me remove the other one.

Reply:
It looks to be from existing implementations. Let me check which one we can remove.